projects/valtimo/user-management/src/lib/user-list/user-list.component.html

```diff
@@ -16,33 +16,43 @@
 
 <div class="main-content pt-0">
   <div class="container-fluid">
-    <div class="text-right">
-      <div class="btn-group mt-m3px mb-3">
-        <button [routerLink]="'create'" class="btn btn-primary btn-space mr-0">
-          <i class="icon mdi mdi-plus mr-1"></i>
-          <span>{{ 'Create new User' | translate }}</span>
-        </button>
+    <div class="row mb-5" *ngIf="userManagementDisabled">
+      <div class="col-12">
+        <div class="alert alert-warning pl-2 plr-2" role="alert">
+          <div class="icon"><span class="mdi mdi-alert-triangle"></span></div>
+          <div class="message" [innerHTML]="'User management not supported' | translate"></div>
```

Comment:
Don't use innerHTML, put the string inside the div tags instead. Also, please work with translation IDs instead of the full string i.e. userManagementNotSupported

Reply:
done